app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ActiveRecord::Base
+  self.per_page = 10
```

Comment:
Can this setting be specified elsewhere? I don't like the fact that the AR model now takes on some responsibility for its HTML presentation.